client/app/models/project.js

```diff
@@ -21,7 +21,7 @@ export default class ProjectModel extends Model {
 
   get pasPackages() {
     const [firstPackage] = this.packages
-      .filter((projectPackage) => projectPackage.dcpPackagetype === 'PAS Package')
+      .filter((projectPackage) => projectPackage.dcpPackagetype === 717170000)
```

Comment:
No lookup hash for this?

Reply:
Hmm, oh yeah I can maybe import the helper here...